bitnami/redis/templates/podmonitor.yaml

```diff
@@ -45,7 +45,7 @@ spec:
       {{- if .honorLabels }}
       honorLabels: {{ .honorLabels }}
       {{- end }}
-      {{- with concat .Values.metrics.podMonitor.relabelings .Values.metrics.podMonitor.relabellings }}
```

Comment:
This is not related to the PR context changes, just a bug fix since the helm context is not correct here

Reply:
moved to separate PR https://github.com/bitnami/charts/pull/25237

Reply:
Thank you for splitting this other fix in an independent PR. The team will review #25237 soon